src/content/editemailsubjectSettings.xul

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?> 
+<?xml-stylesheet href="chrome://global/skin/global.css" type="text/css"?>
+
+<dialog xmlns="http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul"
+		title="EditEmailSubject Settings"
+		id="editemailsubjectSettings"
+		onload="loadSettings()">
```

Comment:
We still have the onload event here, should we addEventListener (into editemailsubjectSettings.js) instead at this stage? Or should this be done when going pure WX?

Reply:
It will be needed, when we turn this into an html page and inline event handlers will be forbidden by CSP. As I want to "group" commits/PRs into conversion steps, I would not like to do it in this PR, even though I did some preperation commits at the beginning of this PR.

The reason for the changed hierachy commit part if this PR is the linkeability of the ConversionHelper in our repo. I knew I wanted to get rid of the extra /chrome/ folder and if I would not have done that now, all links in the upcoming documentation would have been invalidated, as soon as I did it later.